com.mojang.Minecraft.json

```diff
@@ -10,31 +10,18 @@
     "tags": [
         "proprietary"
     ],
-    "sdk-extensions": [
-        "org.freedesktop.Sdk.Extension.openjdk11"
-    ],
-    "add-extensions": {
-        "org.freedesktop.Sdk.Extension.openjdk8": {
-            "directory": "jdk8",
-            "version": "20.08",
-            "no-autodownload": true
-        }
-    },
-    "cleanup-commands": [
-        "mkdir -p /app/jdk8"
-    ],
     "finish-args": [
         "--persist=.minecraft",
         "--socket=x11",
         "--share=ipc",
+        "--socket=wayland",
         "--device=dri",
         "--socket=pulseaudio",
         "--share=network",
-        "--env=JAVA_HOME=/app/jre"
+        "--env=JAVA_HOME=/app/jre",
+        "--filesystem=~/.local/share/flatpak/app/com.mojang.Minecraft:ro",
```

Comment:
What is this (and the next line) for? This most likely broke the Minecraft Flatpak installation, as you mentioned in https://github.com/flathub/com.mojang.Minecraft/pull/83#issuecomment-877695152.

Reply:
This gives the flatpak permission to read the patched glfw file, without this step you can't make use of this PR. Everything else is still fine though without these extra permissions. One if for when installing to your local user, the other for your system.

As I say above under "issues" the way I give permissions here seems... odd. I want to read a bit more to figure out if there's a better way to do this.

Yeah it defintely might have been the cause, I am not sure how exactly since all I'm doing is granting extra permissions.

Also rebasing ended up being very messy thanks to my lack of git ability, so I just made a new PR. https://github.com/flathub/com.mojang.Minecraft/pull/96